src/Tasks.js

```diff
@@ -0,0 +1,85 @@
+let tasksArray = JSON.parse(localStorage.getItem('task')) || [];
+const input = document.querySelector('.input');
+const todoList = document.querySelector('.todo-list');
+
+export default class Task {
+  constructor(description) {
+    this.index = tasksArray.length + 1;
+    this.description = description;
+    this.completed = false;
+  }
+}
+
+const populateList = (tasksArray) => {
+  todoList.innerHTML = '';
+  for (let i = 0; i < tasksArray.length; i += 1) {
+    const li = document.createElement('li');
+    //  li.id = `${tasksArray[i].index}`;
+    li.innerHTML = `<input type="checkbox" class="checkbox" data_id='${i}' ${tasksArray[i].completed ? 'checked' : ''}><div class="task-text ${tasksArray[i].completed ? 'line' : ''}" contenteditable="true" data_id='${i}'>${tasksArray[i].description}</div>
+        <i class="fas fa-trash-alt delete" data_id='${i}'></i>`;
+    todoList.appendChild(li);
+  }
+};
+
+function clearData() {
+  input.value = '';
+}
+
+function updateLS(tasksArray) {
+  localStorage.setItem('task', JSON.stringify(tasksArray));
+}
+
+const updateIndex = () => {
+  for (let i = 0; i < tasksArray.length; i += 1) {
+    tasksArray[i].index = i + 1;
+  }
+};
+
+export function addTask() {
+  if (input.value.trim() !== '') {
```

Comment:
Kindly use a single `export` and include all the function to be exported, instead of repeating export on every function. you can use ```export { function, function} ```